### PR TITLE
Refreshing installation instructions

### PR DESCRIPTION
**Pull request by hyperbolic2346**
Added conjure-up. Updated displays and juju versions to current versions.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7495)

### CODE REVIEW

File: docs/getting-started-guides/ubuntu/installation.md [hunk 1]
```diff
@@ -60,13 +62,26 @@ Bare Metal (MAAS)           | Juju         | Ubuntu | flannel, calico     | [doc
...
 For support level information on all solutions, see the [Table of solutions](/docs/getting-started-guides/#table-of-solutions) chart.
 
-## Configure Juju to use your cloud provider
+## Installation options
+
+You can launch a cluster in one of two ways: [conjure-up](#conjure-up) or [juju deploy](#juju). Conjure-up is just a convenience wrapper over juju and simplifies the installation. As such, it is the preferred method of install.
 
 Deployment of the cluster is [supported on a wide variety of public clouds](#cloud-compatibility), private OpenStack clouds, or raw bare metal clusters. Bare metal deployments are supported via [MAAS](http://maas.io/).
 
+## <a name="conjure-up"></a>Conjure-up
```
Comment (bradtopol) [inline docs/getting-started-guides/ubuntu/installation.md]: This is incorrect and not working properly in the deploy preview.  See above how Installation options is written.  The linking you desired is generated for you if you just follow the similar format

File: docs/getting-started-guides/ubuntu/installation.md [hunk 2]
```diff
@@ -60,13 +62,26 @@ Bare Metal (MAAS)           | Juju         | Ubuntu | flannel, calico     | [doc
...
+To install Kubernetes with conjure-up, you need only to run the following commands and then follow the prompts:
+
+```
+sudo snap install conjure-up --classic
+conjure-up kubernetes
+```
+## <a name="juju"></a>Juju deploy
```
Comment (bradtopol) [inline docs/getting-started-guides/ubuntu/installation.md]: This is incorrect and not working properly in the deploy preview.  See above how Installation options is written.  The linking you desired is generated for you if you just follow the similar format

**Comment by hyperbolic2346** [comment]
Thanks for the help @bradtopol. I have updated the anchors.

**Comment by heckj** [comment]
/approve

:wave: @bradtopol take a look at the changes - if you're good, please add an `/lgtm` to the PR and it'll merge

**Comment by bradtopol** [comment]
Great job!
/lgtm

**Comment by bradtopol** [comment]
Thanks for fixing the merge conflicts!
/lgtm

**Pull request merged**